omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -273,6 +274,19 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 			
 			userValidator.validate(user, errors);
 			
+			if (errors.hasErrors()) {
```

Comment:
Can we combine the logic of this if-clause with the if-clause immediately below it (line 290)?

It looks like the condition is the same and collapsing them into a single if-clause might make the error-handling logic of this section a little more readable.

Reply:
Hi Issac, I have merged in the code as per suggestions.